vispy/visuals/collections/base_collection.py

```diff
@@ -0,0 +1,495 @@
+# -*- coding: utf-8 -*-
```

Comment:
A better name would be basecollection.py (PEP8 discourages underscores in module names)